prow/lib.sh

```diff
@@ -64,7 +64,7 @@ function kind_load_images() {
   for i in {1..3}; do
     # Archived local images and load it into KinD's docker daemon
     # Kubernetes in KinD can only access local images from its docker daemon.
-    docker images "${HUB}/*:${TAG}" --format '{{.Repository}}:{{.Tag}}' | xargs -n1 kind --loglevel debug --name "${NAME}" load docker-image && break
+    docker images "${HUB}/*:${TAG}" --format '{{.Repository}}:{{.Tag}}' | xargs -n1 kind --name "${NAME}" load docker-image && break
```

Comment:
don't like the verbosity? does it hurt?

Reply:
removing it is actually not the same, that's true. I replaced it with `-v9` which is the equivalent on current kind versions